src/pevent_lib.cc

```diff
@@ -55,11 +58,34 @@ static void display_system_config(void) {
   }
 }
 
+int pevent_compute_min_mmap_order(int min_buffer_size_order,
```

Comment:
why not pass the `k_min_number_samples_per_ring_buffer` as argument to the function ?

Reply:
This sounds reasonable indeed.